lib/routes/user/home/payments_filter.dart

```diff
@@ -151,4 +149,37 @@ class PaymentsFilterState extends State<PaymentsFilter> {
     }
     return [PaymentType.RECEIVED, PaymentType.DEPOSIT, PaymentType.SENT, PaymentType.WITHDRAWAL];
   }
+
+  IconButton _buildExportButton(BuildContext context) {
+    return widget._paymentsModel.paymentsList.isNotEmpty
+        ? IconButton(
+            icon: ImageIcon(
+              AssetImage("src/icon/export.png"),
+              color: Colors.white,
+              size: 24.0,
+            ),
+            onPressed: () => _exportPayments(context),
+          )
+        : IconButton(
+            icon: ImageIcon(
+              AssetImage("src/icon/export.png"),
+              color: Theme.of(context).disabledColor,
+              size: 24.0,
+            ),
+          );
+  }
+
+  Future _exportPayments(BuildContext context) async {
+    var action = ExportPayments();
+    widget._accountBloc.userActionsSink.add(action);
+    Navigator.of(context).push(createLoaderRoute(context));
+    action.future.then((filePath) {
+      Navigator.of(context).pop();
+      final RenderBox box = context.findRenderObject();
```

Comment:
Do we need the `RenderBox` here?

Reply:
[No, we don't.](https://github.com/breez/breezmobile/pull/203/commits/e0bd68bfc4b9afe771ee57986d15554dff4ff314)